pkg/targetgroupbinding/targets_manager.go

```diff
@@ -64,6 +68,9 @@ type cachedTargetsManager struct {
 	// chunk size for deregisterTargets API call.
 	deregisterTargetsChunkSize int
 
+	// cidrBlocks specifies the set of IP ranges to watch for in CIDR notation.
+	cidrBlocks []string
```

Comment:
I've deferred parsing at the `New` func since that function signature doesn't return `(*thing, error)` and I didn't want to add more code to plumb down parsing from here: https://github.com/dlmather/aws-load-balancer-controller/blob/5ce8158e71054f2ce5a951e9c13a1685b03548ac/main.go#L180-L200

Reply:
Might be easier w/ the codelink below... hmm

Reply:
Yeah, looks like they also have a helper for parsing: https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/e5d625f96415fd44e6399e9c75e2bd985f5a2288/pkg/networking/utils.go#L10

Reply:
jinx: https://github.com/dlmather/aws-load-balancer-controller/pull/1#discussion_r1653399697

I was hoping the flag parser would support netip.Prefix but unfortunately it doesn't.

Reply:
Just moved the parsing to `func main()`: https://github.com/dlmather/aws-load-balancer-controller/pull/1/commits/46864cf03fbb8658baa07a9eea691464a3b2b622